scripts/npcs/village/priestesslinota.lua

```diff
@@ -261,7 +261,7 @@ local function priestess_talk(npc, ch)
                 .. "I shouldn't leave the shrine unless there's "
                 .. "an emergency. You could prove yourself by going "
                 .. "instead. But I warn you, it could be dangerous.")
-            say("If this doesn't discourage you, I'll explain you "
+            say("If this doesn't discourage you, I'll explain to you "
```

Comment:
Probably the sentence would sound better if "to you" was simply dropped? "I'll explain the issue."

Reply:
I'll change that.